CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
 ### Release highlights
+* Support for adding a `change-id` git header behind the configuration setting `git.change-id`. If this is enabled,
+  `jj` will attempt to use that header to indicate what the change id of the commit should be.
```

Comment:
I don't think most users know what Git commit headers are. Can you rewrite this in a way that's easier for regular users to understand? Maybe something about making change id transfer on `jj git push/fetch`.